SyncIntegrationTests/xcodebuild.py

```diff
@@ -10,7 +10,7 @@
 
 class XCodeBuild(object):
     binary = 'xcodebuild'
-    destination = 'platform=iOS Simulator,name=iPhone 11'
+    destination = 'platform=iOS Simulator,name=iPhone 14'
```

Comment:
We use `iPhone 14` for the XCUITest` so I think that we should use the same here.